pkg/graph/resolvers/mto_solution.go

```diff
@@ -0,0 +1,123 @@
+package resolvers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cms-enterprise/mint-app/pkg/storage/loaders"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cms-enterprise/mint-app/pkg/authentication"
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/storage"
+)
+
+// MTOSolutionUpdate updates the MTOSolution
+func MTOSolutionUpdate(
+	ctx context.Context,
+	logger *zap.Logger,
+	principal authentication.Principal,
+	store *storage.Store,
+	id uuid.UUID,
+	changes map[string]interface{},
+) (*models.MTOSolution, error) {
+	principalAccount := principal.Account()
+	if principalAccount == nil {
+		return nil, fmt.Errorf("principal doesn't have an account, username %s", principal.String())
+	}
+	existing, err := loaders.MTOSolution.ByID.Load(ctx, id)
```

Comment:
We have been following a paradigm of loading methods living in their own function in resolvers unless there is a need to call it explicitly (eg fetching categories).
This  doesn't need to block the PR